.github/actions/build_pandas/action.yml

```diff
@@ -25,8 +25,8 @@ runs:
     - name: Build Pandas
       run: |
         if [[ ${{ inputs.editable }} == "true" ]]; then
-          pip install -e . --no-build-isolation -v
+          pip install -e . --no-build-isolation -v --no-deps
```

Comment:
Curious why do we need the --no-deps flag now?

Reply:
It's needed so numpydev can install dev numpy, otherwise the pin will force pip to install 1.26 there which isn't what we want.